framework/src/main/java/org/checkerframework/framework/type/GenericAnnotatedTypeFactory.java

```diff
@@ -1524,6 +1521,112 @@ protected void performFlowAnalysisForClass(ClassTree classTree) {
     }
   }
 
+  /**
+   * A helper method for {@link #performFlowAnalysisForClass(ClassTree)} that analyzes {@code
+   * method} and all lambdas contained within it.
+   *
+   * @param classTree class tree containing {@code method}
+   * @param method method to analyze
+   * @param classQueue classes found within {@code method} are added to this queue
+   * @param fieldValues values of fields to be used
+   * @param capturedStore the input Store to use for captured variables, e.g. in a lambda
```

Comment:
Is this the input store to the method?  I think so.  The wording makes it seem possible that this is intraprocedural and is used as an input to lambdas.